DOCUMENTION

```diff
@@ -0,0 +1,42 @@
+🚀 TwinTrim Project - Code Explanation ✨
+Welcome to TwinTrim, your go-to tool for efficiently managing duplicate files! Let’s break down the magic behind the scenes 🔍.
```

Comment:
Because I am a curious being who was wondering how only 42 likes could explain everything, here I am. This is literally a readme doc you've created here. 
And to be completely frank, this is even less than readme. Code explanation? I see no explanation simply listing libraries and files.